RAG/chat.py

```diff
@@ -109,6 +108,28 @@ def query(self, query) -> str:
         chain = load_qa_chain(self.__client, chain_type="stuff")
         out = chain.run(input_documents=docs, question=query)
         return out
+    
```

Comment:
Just noting that we will work on these in the future when streaming becomes a priority again